### PR TITLE
added min/max size args for evaluation script

### PR DESCRIPTION
**Pull request by Muhannes**
It was missing for the evaluation script. 
Just copied the way it is done in training script.

### CODE REVIEW

**Comment by hgaiser** [comment]
Awesome, thanks! Could you add it to the COCO evaluation script too while we're at it?

**Comment by Muhannes** [comment]
Sure!

File: keras_retinanet/bin/evaluate.py [hunk 1]
```diff
@@ -91,6 +97,8 @@ def parse_args(args):
     parser.add_argument('--iou-threshold',   help='IoU Threshold to count for a positive detection (defaults to 0.5).', default=0.5, type=float)
     parser.add_argument('--max-detections',  help='Max Detections per image (defaults to 100).', default=100, type=int)
     parser.add_argument('--save-path',       help='Path for saving images with detections.')
+    parser.add_argument('--image-min-side',  help='Rescale the image so the smallest side is min_side.', type=int, default=800)
+    parser.add_argument('--image-max-side',  help='Rescale the image if the largest side is larger than max_side.', type=int, default=1333)
```
[the remark this answers is not in the document]
Comment (Muhannes) [reply keras_retinanet/bin/evaluate.py]: The help is the same as it was for the train script. So if that is the case, then it is wrong there too.

[the remark this answers is not in the document]
Comment (hgaiser) [reply keras_retinanet/bin/evaluate.py]: Ah yeah, seems like it.

[the remark this answers is not in the document]
Comment (Muhannes) [reply keras_retinanet/bin/evaluate.py]: Should i change it, or fix it in another PR?

[the remark this answers is not in the document]
Comment (hgaiser) [reply keras_retinanet/bin/evaluate.py]: If you could change it, that'd be awesome :)

[the remark this answers is not in the document]
Comment (hgaiser) [reply keras_retinanet/bin/evaluate.py]: Did you change it? I think I misinterpreted what it said, it's actually correct as is.

**Comment by hgaiser** [comment]
Thanks :+1:

File: keras_retinanet/bin/evaluate.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (Muhannes) [reply keras_retinanet/bin/evaluate.py]: No i did not change it as i suppose u noticed.

**Pull request merged**